core/client/fs/src/main/java/alluxio/util/FileSystemOptionsUtils.java

```diff
@@ -140,17 +140,19 @@ public static CreateFilePOptions createFileDefaults(AlluxioConfiguration conf, b
    * @return options based on the configuration
    */
   public static DeletePOptions deleteDefaults(AlluxioConfiguration conf) {
```

Comment:
seems like this change the semantic of delete in baseFileSystem. Is this discussed with David?

Reply:
This was not discussed with David, I will make the change without changing the semantics.